### PR TITLE
Add ignore flags

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️

### CODE REVIEW

File: src/Debugger.php [hunk 1]
```diff
@@ -98,7 +98,10 @@ private function isRequestIgnored(ServerRequestInterface $request): bool
                 return true;
             }
         }
-        return false;
+        return (bool) ($request->hasHeader('X-Debug-Ignore') && $request->getHeaderLine('X-Debug-Ignore') === 'true')
```
Comment (vjik) [inline src/Debugger.php:101]: Should we use Yii prefix? 

`X-Yii-Debug-Ignore`

File: src/Debugger.php [hunk 2]
```diff
@@ -98,7 +98,10 @@ private function isRequestIgnored(ServerRequestInterface $request): bool
...
+
+
+         ;
```
Comment (vjik) [inline src/Debugger.php:104]:
```suggestion
        return $request->getHeaderLine('X-Debug-Ignore') === 'true';
```

When header not exist `getHeaderLine()` returns empty string.

File: src/Debugger.php [hunk 3]
```diff
@@ -111,7 +114,10 @@ private function isCommandIgnored(?string $command): bool
                 return true;
             }
         }
-        return false;
+        return (bool) (getenv('YII_DEBUG_IGNORE') === 'true')
+
+
+        ;
```
Comment (vjik) [inline src/Debugger.php:120]:
```suggestion
        return getenv('YII_DEBUG_IGNORE') === 'true';
```

**Review by vjik — CHANGES_REQUESTED** [review]

File: src/Debugger.php (hunk 1 above)
Comment (xepozz) [reply src/Debugger.php:101]: Nope

File: src/Debugger.php (hunk 2 above)
Comment (xepozz) [reply src/Debugger.php:104]: Then the condition turns to `false`?

Comment (xepozz) [reply src/Debugger.php]: That's what this needs

File: src/Debugger.php [hunk 4]
```diff
@@ -111,7 +114,7 @@ private function isCommandIgnored(?string $command): bool
                 return true;
             }
         }
-        return false;
+        return getenv('YII_DEBUG_IGNORE') === 'true';
```
Comment (vjik) [inline src/Debugger.php:117]: Seems this code should be first in this method.

File: src/Debugger.php (hunk 1 above)
Comment (xepozz) [reply src/Debugger.php:101]: We also don't use the prefix in the headers middleware
https://github.com/yiisoft/yii-debug-api/blob/master/src/Middleware/DebugHeaders.php#L30

Comment (vjik) [inline src/Debugger.php:101]: Seems this code should be first in this method.

**Pull request merged**